config/gregtech/worldgen/overworld/natural_gas_plate.json

```diff
@@ -1,32 +1,43 @@
 {
-  "weight": 25,
-  "density": 1.0,
-  "max_height": 50,
-  "min_height": 10,
-  "priority": -100,
-  "count_as_vein": false,
-  "generation_predicate": "any",
-
-  "biome_modifier": {
-    "type": "biome_dictionary",
-    "sandy": 5
-  },
-  "generator": {
-    "type": "plate",
-    "length": [20, 36],
-    "depth": [20, 36],
-    "height": [5, 10]
-  },
-  "filler": {
-    "type": "ignore_bedrock",
-    "value": {
-      "type": "weight_random",
-      "values": [
-        {
-          "weight": 100,
-          "value": "fluid:natural_gas"
+    "weight": 25,
+    "density": 1.0,
+    "max_height": 50,
+    "min_height": 10,
+    "priority": -100,
+    "count_as_vein": false,
+    "generation_predicate": "any",
+    "biome_modifier": {
+        "type": "biome_dictionary",
+        "sandy": 5
+    },
+    "generator": {
+        "type": "plate",
+        "length": [
+            20,
+            36
+        ],
+        "depth": [
+            20,
+            36
+        ],
+        "height": [
+            5,
+            10
+        ]
+    },
+    "filler": {
+        "type": "ignore_bedrock",
+        "value": {
+            "type": "weight_random",
+            "values": [
+                {
+                    "weight": 100,
+                    "value": "fluid:natural_gas"
```

Comment:
I have not once ever found one of these veins. Are they even worth including?

Reply:
they were there so i kept it